lib/node/dropIndex.ts

```diff
@@ -21,6 +21,11 @@ export class DropIndexNode extends Node {
         }
 
         this.table = table;
-        this.options = { indexName };
+        this.options = { indexName, ifExists: false };
```

Comment:
please remove default false value as this new property will only have effect when true